### DISCUSSION TITLE
[Feature]: Minor Release for String Assertion Improvement

### DISCUSSION THREAD

**Discussion by mrafnadeem-apimatic** [fluentassertions/fluentassertions#2667] [General]
### Background and motivation

Our projects extensively manipulate big strings. When writing tests, it can often be difficult to see where a change is made. I see that #2307 improves this enormously. But I noticed that this is only available in the v7 alpha pre-release. Is it not possible to release a stable minor version with this improvement? It saves a whole lot of time, especially when whitespace characters cause tests to fail in big strings.

### Are you willing help with a pull-request?

No

**Comment by ITaluone** [comment]
You have two other possibilities here:

1) Use the latest alpha (alpha.4)
2) compile the lastet develop yourself

**Comment by dennisdoomen** [comment]
Unfortunately, we can't. v7 contains a lot of breaking changes which will need to be part of a major release to comply to the ideas behind Semantic Versioning.
